CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Origin (DCO). This document was created by the Linux Kernel community and is a
 simple statement that you, as a contributor, have the legal right to make the
 contribution. See the [DCO](DCO) file for details.
 
-Any copyright notices in this repo should specify the authors as "the Loodse XXX project contributors".
+Any copyright notices in this repo should specify the authors as "the Loodse machine-controller project contributors".
```

Comment:
The current copyright header says `Copyright 2019 The Machine Controller Authors.`. 

Can you change the template files in `boilerplate` dir to reflect this? https://github.com/kubermatic/machine-controller/blob/master/hack/boilerplate/boilerplate.go.txt#L2

Reply:
Sure. So I also can add the missing "Signoff". 😅